ts/modules/networkgraph/draggable-nodes.ts

```diff
@@ -0,0 +1,253 @@
+/* *
+ *
+ *  Networkgraph series
```

Comment:
Actually, PackedBubble series also uses this module. Should move this to mixins or modules?

Reply:
Can we delay this until after this migration phase?

Reply:
Yes, of course! I'm just thinking out loud :)